src/haz3lweb/view/SchoolMode.re

```diff
@@ -51,6 +52,11 @@ let render_cells = (settings: ModelSettings.t, v: list(vis_marked(Node.t))) => {
   );
 };
 
+/* Generates the DOM objects for each component
```

Comment:
view is fairly self-explanatory throughout the codebase so no documentation needed here